helm_deploy/prisoner-content-hub-backend/values.yaml

```diff
@@ -73,3 +73,9 @@ ingress:
     nginx.ingress.kubernetes.io/server-snippet: |
        add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
     nginx.ingress.kubernetes.io/proxy-body-size: 500m
+
+cron:
+  schedule: "*/* 3 * * *"
+  image:
+    repository: byrnedo/alpine-curl
```

Comment:
:+1: for the `alpine` image. Could/should we publish our own image for this? This is a personal image, so could be swapped out for something more nefarious or fall out of maintenance.